app/src/js/components/FormRequest/overview.js

```diff
@@ -53,17 +68,50 @@ const FormsOverview = ({ forms, requests }) => {
     setSelected(selectedCopy);
   };
 
+  const showAlert = (message, variant, dissmissTimer) => {
+    setAlertVariant(variant);
+    setAlertMessage(message);
+    setDismissCountDown(dissmissTimer);
+    window.scrollTo(0,0)
+  }
+
   const submitForm = async (e) => {
     e.preventDefault();
     const { basepath } = _config;
     const urlReturn = `${basepath}requests`;
 
     if (requestId){
       try {
-        await dispatch(assignDaacs({'id': requestId, 'daacs': selected}))
-        window.location.href = urlReturn;
+        // If the submission requires additional DAAC review make sure that it's only being assigned to 1 DAAC
+        // At the time of this implementation it was assumed that if an additional review was needed it would only go to one DAAC at a time.
```

Comment:
Great catch here! I'm sure it will also be more transparent if this assumption turns out not to be true.